README.md

```diff
@@ -186,6 +192,37 @@ a common baseline. You can find a complete example [here](./examples/templates/c
 Using the `--mark-processed` option alongside `--json-template` will mark all resources as `bootstrapProcessed`,
 which will make it so `update_space` calls avoid already created resources. (A resource being either a Content Type, Entry or Asset).
 
+## Workflow for backing up draft/updated content
+
+On many cases, you want to have a full dump of your whole space, including draft/updated content.
+To achieve this, do the following steps:
```

Comment:
minimal nit, I think it should be ", complete the following steps:"

I don't think you "do"steps, am I right @loudmouth?

Reply:
how about `follow the next steps`?

Reply:
Just noticed it should be "In many cases"... missed that before, sorry 😞

Reply:
"do the following"